### DISCUSSION TITLE
Build error on BPIR4 Firmware with imagebuilder

### DISCUSSION THREAD

**Discussion by mdsdtech** [immortalwrt/immortalwrt#1558] [Q&A] [answered]
Description:
While attempting to build firmware using imagebuilder for the Banana Pi R4 (BPIR4), the build process fails with an error indicating that the mt7988-wo-firmware package cannot be installed. This causes the overall make process to terminate with an error.

Error Output:
```
Collected errors:
 * opkg_install_cmd: Cannot install package mt7988-wo-firmware.
make[2]: *** [Makefile:228: package_install] Error 255
make[1]: *** [Makefile:155: _call_image] Error 2
make: *** [Makefile:331: image] Error 2
```

Steps to Reproduce

1. Run the build process for the BPIR4 firmware using make command:
`make image PACKAGES="luci luci-app-firewall luci-app-opkg luci-base luci-compat" PROFILE="bananapi_bpi-r4"`
2. The build process fails with the error above when attempting to install the mt7988-wo-firmware package.

**Comment by 1715173329** [comment, marked as answer]
Depends on buildbot update. You can trim this for now.

**Reply by mdsdtech** [reply]
Got it, thanks!
